src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -63,7 +65,25 @@ func (manager *Manager) GetSnapshotId(desiredName string) (string, error) {
 
 // ValidateName - does syntactic validation on the name
 func (manager Manager) ValidateName(name string) error {
-	// validate name
+	if len(name) == 0 {
+		return fmt.Errorf("a snapshot name may not be the empty string")
+	}
+	reportedName := name
+	if len(reportedName) > nameDisplayCutoffSize {
+		reportedName = reportedName[0:nameDisplayCutoffSize] + "..."
+	}
+	if len(name) > maxNameLength {
+		return fmt.Errorf(`invalid name %q: max length is %d, %d were specified`, reportedName, maxNameLength, len(name))
+	}
+	if isWhitespaceRegexp.MatchString(name[0:1]) {
+		return fmt.Errorf(`invalid name %q: may not start with a white-space character`, reportedName)
```

Comment:
Use "must not" instead of "may not".

When possible use RFC terminology: "must" (required), "must not", "should" (recommended), "should not", and "may" (optional).